docs/api/python/gluon/contrib.md

```diff
@@ -80,6 +80,20 @@ In the rest of this document, we list routines provided by the `gluon.contrib` p
     WikiText103
 ```
 
+#### Parallel
+
+```eval_rst
+.. currentmodule:: mxnet.gluon.parallel
+
+.. autosummary::
+    :nosignatures:
+    
+    DataParallelModel
+    DataParallelCriterion
```

Comment:
This make it easier for the users, because the situation is complicated for network with multiple output cases. We just handle the situation internally.

Reply:
I think this is too many API's and prone to misuse.

Also we use the term Loss instead of Criterion

Reply:
Will change the name :)